deps/build.jl

```diff
@@ -12,9 +12,16 @@ download(
 const tmp = joinpath(_pkg_assets, "tmp")
 !isdir(tmp) && mkdir(tmp)
 
-download("https://github.com/KaTeX/KaTeX/releases/download/v0.10.0/katex.tar.gz", joinpath(tmp, "katex.tar.gz"))
+const download_dir = joinpath(_pkg_assets, "download")
```

Comment:
Could the download also happen inside `tmp`? Maybe it can be split as `tmp/download` and `tmp/extract`. I find it cleaner to just remove one `tmp` folder at the end